config/hub/samples/clustermanager_v1alpha1_singleclusterworkload.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: clustermanager.onprem.openshift.io/v1alpha1
+kind: SingleClusterWorkload
+metadata:
+  name: singleclusterworkload-sample
+spec:
+  bagOfYAML: |-
+   --- 
+   kind: Deployment
+   apiVersion: extensions/v1beta1
+   metadata:
+     name: nginx-deployment
+   spec:
+     replicas: 3
+     strategy: Recreate
+     selector:
+       matchLabels:
+         deploy: example
+     template:
+       metadata:
+         labels:
+           deploy: example
+       spec:
+         containers:
+           - name: nginx
+             image: nginx:1.7.9
+    ---
+  serviceaccount: gcp-spoke-serviceaccount
```

Comment:
Note: the `serviceaccount` field for the SingleClusterWorkload CR will be populated from the `serviceaccount` field in the MultiClusterWorkload CR by the hub controller when it creates the individual SingleClusterWorkload CR instances. Also, the namespace for the generated CRs will be the same as the namespace of the MultiClusterWorkload CR by default.